.github/workflows/bot-mere-denis.yml

```diff
@@ -70,3 +70,5 @@ jobs:
           SEED: ${{ secrets.SEED1 }}
           BOT_ENVIRONMENT: testing
           BOT_FILTER_FAMILY: ${{github.event.inputs.family}}
+          ALLURE_LOGIN: ${{secrets.ALLURE_LOGIN}}
+          ALLURE_PASSWORD: ${{secrets.ALLURE_PASSWORD}}
```

Comment:
why only mere denis? we have 7 bots and mere denis isn't used very often.

Reply:
just an example for @nabil-brn
this was just a commit to help him set things up, the PR is not ready yet

Reply:
Imo we can't mix up reports from different "bot philosophy". The report history won't be consistent.
I think we can monitor all of them but in a separate "space", I gonna do some change.

Thank you for review